inc/classes/optimization/CSS/class-abstract-css-optimization.php

```diff
@@ -119,7 +119,7 @@ protected function is_minify_excluded_file( $tag ) {
 			return true;
 		}
 
-		$file_path = rocket_extract_url_component( $tag[2], PHP_URL_PATH );
+		$file_path = wp_parse_url( $tag[2], PHP_URL_PATH );
```

Comment:
1. Type hint the `array $tag` in the function params list.
2. What if `$tag` is empty? There wouldn't be a `0` and `2` index, which would cause a notice/warning. We need to protect this method.

Reply:
Done in https://github.com/wp-media/wp-rocket/commit/dc6eb9dbc5ad6671d69ac3cbe71d796dc5da58f7